deployment/data/nginx/app.conf

```diff
@@ -0,0 +1,62 @@
+upstream core_backend {
+    server core_backend:8000 fail_timeout=0;
+}
+
+upstream admin_app {
+    server admin_app:3000 fail_timeout=0;
+}
+
+server {
+    listen 80;
+    server_name localhost;
+
+    location ~ ^/api/(.*)$ {
```

Comment:
So all call with path `/api/(something)` are redirected to `core_backend/(something)`. The `/api` bit is stripped.